resources/views/platform/components/game/achievements-list/root.blade.php

```diff
@@ -2,24 +2,21 @@
     'achievements' => [],
     'beatenGameCreditDialogContext' => 's:|h:',
     'totalPlayerCount' => 0,
-    'progressionTypeValue' => 'progression', // `AchievementType`
-    'winConditionTypeValue' => 'win_condition', // `AchievementType`
     'isCreditDialogEnabled' => true,
     'showAuthorNames' => false,
 ])
 
 <?php
+use App\Platform\Enums\AchievementType;
```

Comment:
Is this needed? It looks like all the other changes in the file removed code.

Reply:
Unneeded. Removed in latest.